File: Practice(Beginner)/Python3/Area_OR_Perimeter.py
# Source: https://www.codechef.com/problems/AREAPERI

l = int(input())
b = int(input())

if(l*b > 2*(l+b)):
    print("Area\n"+str(l*b))
elif(l*b < 2*(l+b)):
    print("Peri\n"+str(2*(l+b)))
else:
    print("Eq\n"+str(l*b))

File: Practice(Beginner)/C++/Area OR Perimeter.cpp
// Source: https://www.codechef.com/problems/AREAPERI

#include <bits/stdc++.h>
using namespace std;

int main() {
    int l, b;
    cin >> l >> b;
    l*b == 2*(l+b) ? cout << "Eq\n" << l*b : l*b > 2*(l+b) ? cout << "Area\n" << l*b : cout << "Peri\n" << 2*(l+b);
}